cli/cmd/policy.go

```diff
@@ -125,6 +126,15 @@ func init() {
 	policyCmd.AddCommand(policyShowCmd)
 	policyCmd.AddCommand(policyDeleteCmd)
 
+	// Lacework Content Library
+	if IsLCLInstalled(*cli.LwComponents) {
```

Comment:
This looks a bit trippy `IsLCLInstalled` 👁️ 👁️  - I thought you were checking, from the CLI, if the CLI was installed. 😂

Reply:
I've done quite a few double takes on this myself!